packages/api-openfin/src/shims.ts

```diff
@@ -1,16 +1,22 @@
-window.Notification = function(title, options) {
-  if (!options) {
-    options = {};
-  }
+interface Window {
```

Comment:
It's probably worth moving this comment, or something similar, into the `shim.ts` file itself.